### PR TITLE
fix(Slide): initialize max-height to work SSR

### PR DESCRIPTION
**Pull request by silvenon**
We were setting the `maxHeight` state in `componentDidMount`, and
considering that this method only runs in the browser, `max-height`
ended up being `0` in server-side rendering. There is no reason not to
initialize the state with the `maxHeight` prop right away.
<br/><br/><br/><url>LiveURL: https://orbit-fix-slide-expanded-ssr.surge.sh</url>

### CODE REVIEW

**Comment by silvenon** [comment]
Forgot to complete tests, hold on…

File: packages/orbit-components/src/utils/Slide/index.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ StyledSlide.defaultProps = {
 
 class Slide extends React.Component<Props, State> {
   state = {
-    maxHeight: 0,
+    maxHeight: typeof this.props.maxHeight !== "undefined" ? this.props.maxHeight : null,
```
Comment (mainframev) [inline packages/orbit-components/src/utils/Slide/index.js:31]: should it be null? or if false set to `0` maybe?

Comment (silvenon) [reply packages/orbit-components/src/utils/Slide/index.js]: Yep!

**Comment by silvenon** [comment]
@mainframev fixed, ready for review 👍

**Review by mainframev — APPROVED** [review]

**Pull request merged**